src/librustc/lint/builtin.rs

```diff
@@ -337,6 +337,12 @@ declare_lint! {
     "failures in resolving intra-doc link targets"
 }
 
+declare_lint! {
+    pub NO_CRATE_LEVEL_DOC,
```

Comment:
I think `MISSING_CRATE_LEVEL_DOC` would be a be a bit better in communicating what's wrong. Also it reads more fluidly when used with `allow/warn/deny` (see [RFC-0344#lints](https://rust-lang.github.io/rfcs/0344-conventions-galore.html#lints))

Reply:
:+1: